Neos.Neos/Classes/View/FusionView.php

```diff
@@ -73,16 +76,13 @@ public function render(): ResponseInterface|StreamInterface
         $currentNode = $this->getCurrentNode();
 
         $subgraph = $this->contentRepositoryRegistry->subgraphForNode($currentNode);
-        $currentSiteNode = $subgraph->findClosestNode($currentNode->aggregateId, FindClosestNodeFilter::create(nodeTypes: NodeTypeNameFactory::NAME_SITE));
-
-        if (!$currentSiteNode) {
-            throw new \RuntimeException('No site node found!', 1697053346);
-        }
-
+        $currentSiteNode = $this->getCurrentSiteNode();
         $fusionRuntime = $this->getFusionRuntime($currentSiteNode);
 
         $this->setFallbackRuleFromDimension($currentNode->dimensionSpacePoint);
 
+        $this->nodeTypeManager = $this->contentRepositoryRegistry->get($subgraph->getContentRepositoryId())->getNodeTypeManager();
```

Comment:
It seems a bit weird to me that we instantiate the NTM in the `render()` method.
That makes the behavior a bit fragile because it will throw when accessed before the render method was called once

Reply:
I simplified the code and get the NTM when we need it.